draft-ietf-rats-msg-wrap.md

```diff
@@ -282,18 +289,30 @@ Since the collection type is recursive, implementations may limit the allowed de
 {: #fig-cddl-collection artwork-align="left"
    title="CDDL definition of the CMW collection format"}
 
-Although initially designed for the composite Attester use case, the CMW collection can be repurposed for other use cases requiring CMW aggregation.
+CMW itself provides no facilities for authenticity, integrity protection, privacy or attestation security.
+It is up to the designer of each use case to understand the security properties necessary and add them around the collection.
+A secure channel (e.g., using TLS) or some form of object security (e.g., using JWT) may be sufficient, but not always.
+
+When a CMW is used to carry the Evidence for composite or layered attestation for a single device, the security properties needed are that of attestation.
+In particular, all the members in a CMW must be bound together so that an attacker can not replace one Evidence message showing compromise with that from a non-compromised device.
+The authenticity and integrity protection MUST be attestation-oriented.
```

Comment:
I'm not sure what these lines are compelling me to do. Why aren't lines 292 - 294 sufficient to convey that expectation that security of the CMW isn't part of this document? The normative wording seems to contradict the previous paragraph. It seems to acknowledge an additional security consideration that an active attacker could drop portions of the collection - which is an attack that is reasonably covered in lines 292-294.

Reply:
This captures the third point from [Laurence's email](https://mailarchive.ietf.org/arch/msg/rats/hMz-UQMchfmGeXq7saLWToXvndk/):

> _"The third thing is discussion of security when the use case is attestation Evidence, in which case the full attestation security model is necessary (which is more than integrity protection or a CWT)."_

Reply:
The wording seems awkward. Is it different from: "A CMW collection is subject to a variety of security threats such as insertion / removal of CMW contents, replay, misdirection, snooping, etc."

Reply:
Since CMW doesn't define security mechanisms, it seems impossible for an implementer to satisfy the MUST requirement. I'm not sure what "attestation oriented" means. It reads like normative on some other spec besides this spec.

Reply:
> Since CMW doesn't define security mechanisms, it seems impossible for an implementer to satisfy the MUST requirement.

I agree that the MUST here is unenforceable.  I'll remove it and rephrase Laurence's point to keep the same "strength of signal" without bothering RFC2119. 

> I'm not sure what "attestation oriented" means. It reads like normative on some other spec besides this spec.

He means obeying https://www.rfc-editor.org/rfc/rfc9334.html#section-12.1

Reply:
Please see https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/118/commits/5c5928ada08d417c74e6321581bf2a0d8d3aab56